File: day15.java


    public static  Node insert(Node head,int data) {
        //Complete this method
        Node newNode = new Node(data);
        Node current = head;
        if(current == null){
            return newNode;
        }
        if(current.next == null){
            current.next = newNode;
            return head;
        }
        while(current.next != null){
            current = current.next;
        }
        current.next = newNode;
        return head;
    }

File: day15.py


    def insert(self,head,data):
        newNode = Node(data)
        current = head
        if(current == None):
            return newNode;
        if(current.next == None):
            current.next = newNode;
            return head;
        while(current.next != None):
            current = current.next;

        current.next = newNode;
        return head;
